src/styles/ToggleStyles.ts

```diff
@@ -13,8 +13,7 @@ export const ToggleContainer = styled.div`
 	justify-content: center;
 	align-items: center;
 	font-weight: normal;
-	font-size: 1.2vw;
-`;
+	font-size: min(1.5vw, 20px);`;
```

Comment:
jesli chcesz, żeby to dotyczyło wszystkich przycisków (spójność rozmiarów czcionek) to przerzuć to do BaseButton